source/fundamentals/crud/read-operations/cursor.txt

```diff
@@ -58,6 +58,21 @@ results:
 This method is often used when your query filter will match one
 document, such as when filtering by a unique index.
 
+Available
+~~~~~~~~~
+
+Use the ``available()`` method to retrieve the amount of results
+available locally without blocking:
+
+.. literalinclude:: /includes/fundamentals/code-snippets/Cursor.java
+   :language: java
+   :dedent:
+   :start-after: begin availableExample
+   :end-before: end availableExample
+
+The method returns ``0`` when the ``MongoClient`` exhausts the cursor or
+if the cursor is closed.
```

Comment:
Since this page is specific information about a Cursor (and there doesn't seem to be any other page that describes them), I wonder if it could be useful to describe what it means to "exhaust" a cursor. 

If the intent is for the reader to learn the term, maybe providing a definition could be helpful. Otherwise, you might try describing it without using the word "exhaust" since that's terminology we might not expect someone to know if they don't know much about cursors.